### PR TITLE
EPMDJ-9248: Add name for all tables

### PR DESCRIPTION
**Pull request by dmitrii-kliushentsev**

### CODE REVIEW

File: src/pages/agent/methods-table/methods-table.tsx [hunk 1]
```diff
@@ -275,6 +284,7 @@ export const MethodsTable = ({
             message="Try adjusting your search or filter to find what you are looking for."
           />
         )}
+        {...renderProps}
```
Comment (RomanAchilov) [inline src/pages/agent/methods-table/methods-table.tsx]:
```suggestion
    renderheader={(props) => <Component {...props}/>}
```

File: src/pages/agent/tests-table/test-details.tsx [hunk 2]
```diff
@@ -121,11 +130,13 @@ export const TestDetails = ({
     )), [tests.length]);
 
   return (
-    <div tw="flex flex-col mt-12" data-test="test-details:table-wrapper">
+    <div tw="flex flex-col pt-8" data-test="test-details:table-wrapper">
```
Comment (RomanAchilov) [inline src/pages/agent/tests-table/test-details.tsx]: mt

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/pages/agent/tests-to-run/tests-to-run.tsx [hunk 3]
```diff
@@ -163,6 +160,12 @@ export const TestsToRun = ({ agentType = "Agent" }: Props) => {
                 accessor: "overview.duration",
                 Cell: ({ value, row: { original: { toRun } } }: any) => (toRun ? null : <Cells.Duration value={value} />),
               }]}
+            renderHeader={(currentCount: number) => (
+              <div tw="flex justify-between text-monochrome-default text-14 leading-24 pb-3">
+                <div tw="uppercase font-bold">{`All suggested tests (${currentCount})`}</div>
+                <div />
```
Comment (IliaSolovev) [inline src/pages/agent/tests-to-run/tests-to-run.tsx]: remove

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

**Review by IliaSolovev — APPROVED** [review]

File: src/pages/agent/tests-to-run/tests-to-run.tsx (hunk 3 above)
Comment (IliaSolovev) [reply src/pages/agent/tests-to-run/tests-to-run.tsx]:
```suggestion
               
```

**Pull request merged**